Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -93,6 +93,14 @@ public int RemainingItemsThreshold
 			set => SetValue(RemainingItemsThresholdProperty, value);
 		}
 
+		public static readonly BindableProperty ItemChangeAnimationsEnabledProperty = BindableProperty.Create(nameof(ItemChangeAnimationsEnabled), typeof(bool), typeof(ItemsView), true);
```

Comment:
should `true` be the default ? I understood that the purpose of this was to enable animations...